ci/roundtrip_local_to_c8y.py

```diff
@@ -109,25 +120,33 @@ def check_timestamps(timestamps, laststamp):
 
     failed = False
 
+    warning = 0
+
+    assert is_timezone_aware(laststamp)
+
     for tstamp in timestamps:
-        # All timestamps seem to be in UTC time -> will end with 'Z'
-        # fromisoformat does not seem to cope with the Z, so we remove it
 
         if tstamp.endswith("Z"):
+            # fromisoformat does not seem to cope with the Z, so we remove it
+            # Workaround: remove the Z and make the timestamp UTC aware
+            #
+            # Alternativelyuse:
+            # dateutil.parser.isoparse is available in the third-party package dateutil.
+            # https://dateutil.readthedocs.io/en/stable/parser.html#dateutil.parser.isoparse
+
             tstamp = tstamp[:-1]
-        else:
-            print("Timestamp verification: Z is missing")
-            failed = True
+            tstamp += "+00:00"
```

Comment:
Surprised to see that this prefix is added unconditionally. What if there is already a time offset?

Reply:
My bad ... I missed the fact that the `else` was commented out.